ckan/config/middleware/flask_app.py

```diff
@@ -502,6 +502,9 @@ def _register_error_handler(app):
 
     def error_handler(e):
         debug = config.get_value('debug')
+        if request.path.endswith('/'):
+            site_url = config.get_value('ckan.site_url')
+            return helpers.redirect_to(site_url + request.path[:-1])
```

Comment:
I suspect this won't work on sites with a [root_path](https://docs.ckan.org/en/2.9/maintaining/configuration.html#ckan-root-path) setting. Also you could trigger lots of redirects with silly urls like https://mysite.example.com///////////////////////...

Reply:
@wardi thanks again for the help I totally forgot about that condition with ////// 
I can fix that problem with strip/regex, and regarding `ckan.rooth_path`, to be honest, I've never used it, but I will test it and will figure out the best option.